distributed/scheduler.py

```diff
@@ -789,6 +789,11 @@ class TaskPrefix:
 
        An exponentially weighted moving average duration of all tasks with this prefix
 
+    .. attribute:: suspicious: int
+
+       Numbers of tasks marked as suspicious with this prefix
```

Comment:
```suggestion
       Numbers of times a task was marked as suspicious with this prefix
```

A single task might add a few to this count